### PR TITLE
375 refactor legend

### PR DESCRIPTION
**Pull request by mmazanec22**
Realized I needed the changes I had begun to make for #375 in order to complete #393 anyway, so I added the missing info counts in this refactor.

### CODE REVIEW

File: src/Hedwig/ClientApp/src/assets/styles/_uswds-theme-custom-styles.scss [hunk 1]
```diff
@@ -515,9 +515,13 @@ $future-step-color: color('base-dark') !default;
...
 	position: relative;
-	top: -0.1em;
-	left: 0.1em;
-	margin-right: 0.25em;
+	height: 100%;
+	vertical-align: initial;
+	margin-right: 0.5em;
+}
+
+.oec-legend__symbol ~ * {
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/assets/styles/_uswds-theme-custom-styles.scss]: Ooh what's this CSS selector do?

File: src/Hedwig/ClientApp/src/containers/Enrollment/Detail/EnrollmentDetail.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ export default function EnrollmentDetail({
 	}
 
 	const child = enrollment.child;
-	const informationIsMissing = missingInformation(enrollment);
+	const informationIsMissing = isIncompleteEnrollment(enrollment);
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/containers/Enrollment/Detail/EnrollmentDetail.tsx]: I don't think we want to make these changes here. @jlhogan has a PR out that re-does a lot of this missing/incomplete work.

File: src/Hedwig/ClientApp/src/containers/Roster/Roster.tsx [hunk 3]
```diff
@@ -55,17 +61,6 @@ export default function Roster() {
 		return <div className="Roster"></div>;
 	}
 
-	// Note: These explicit is(In)CompleteEnrollment functions is necessary due to Typescript limitations
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/containers/Roster/Roster.tsx]: I also just put out a PR #448 that removes these and adds some better typings elsewhere fyi

File: src/Hedwig/ClientApp/src/utils/enrollmentCompletenessUtils.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { Enrollment } from '../generated/models/Enrollment';
```
Comment (cailyncodes) [inline src/Hedwig/ClientApp/src/utils/enrollmentCompletenessUtils.ts]: See earlier note about these being removed in PR #448

File: src/Hedwig/ClientApp/src/assets/styles/_uswds-theme-custom-styles.scss (hunk 1 above)
Comment (mmazanec22) [reply src/Hedwig/ClientApp/src/assets/styles/_uswds-theme-custom-styles.scss]: Oh hmm I think this is a leftover from an experiment that was a bad idea... but it's grabbing siblings

**Pull request merged**